test/phlex/view/text.rb

```diff
@@ -3,6 +3,19 @@
 describe Phlex::HTML do
 	extend ViewHelper
 
+	with "deprecated text method" do
+		view do
+			def template
+				text "Depreacted"
+			end
+		end
+
+		it "renders content and prints deprecation warning" do
+			expect(example).to receive(:warn).with("DEPRECATED: The `text` method has been deprecated in favour of `plain`. Please use `plain` instead. The `text` method will be removed in a future version of Phlex. Called from: test/phlex/view/text.rb:9:in `template'")
+			expect(output).to be == "Depreacted"
+		end
+	end
+
```

Comment:
We can remove this test and the `text` method in this PR if you want. It needs to go in the next minor release anyway.